csharp-advanced-topics/GenericsInCSharp/GenericsInCSharp/Program.cs

```diff
@@ -11,9 +11,10 @@ static void Main(string[] args)
             var multiplication = new ArithmeticDelegates<int>(delegateObj.MultiplicationFunc);
             var num1 = 5;
```

Comment:
Could you add a single line space here please? ☺️

Reply:
ooh got it, fixing it shortly